src/MUIDataTable.js

```diff
@@ -170,6 +112,7 @@ class MUIDataTable extends React.Component {
       serverSide: false,
       rowHover: true,
       fixedHeader: true,
+      fixedToolbar: true,
```

Comment:
Should this be the default? Breaks backwards compat, but I think it's what people expect

Reply:
That's fine I would expect this to be default